keps/sig-api-machinery/4008-crd-ratcheting/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: CRD Validation Ratcheting
+kep-number: 4008
+authors:
+  - "@alexzielenski"
+owning-sig: sig-api-machinery
+participating-sigs:
+status: implementable
+creation-date: 2023-05-15
+reviewers:
+  - "@jpbetz"
+  - "@apelisse"
+  - '@stts'
```

Comment:
sttts, 3 ts

Reply:
fixed. apologies for the oversight

Reply:
No worries. stts got used to it I guess. Happens often 😀